### ISSUE TITLE
WindowsFileSystems.getFileStores() doesn't report mounted volumes

### ISSUE THREAD

**Issue by henryx** [dblock/oshi#169]
If we have a similar situation:
```
C:\> wmic diskdrive list brief
Caption                   DeviceID            Model                     Partitions  Size
VBOX HARDDISK ATA Device  \\.\PHYSICALDRIVE0  VBOX HARDDISK ATA Device  2           42944186880
VBOX HARDDISK ATA Device  \\.\PHYSICALDRIVE1  VBOX HARDDISK ATA Device  1           21467980800

C:\> wmic volume list brief
Capacity     DriveType  FileSystem  FreeSpace    Label                     Name

104853504    3          NTFS        75137024     Reserved for the system   \\?\Volume{a60e769e-9c37-11e2-a220-806e6f6e6963}\
21471686656  3          NTFS        21376950272  Volume                    C:\mount\
42842714112  3          NTFS        11597103104                            C:\
58157056     5          CDFS        0            VBOXADDITIONS_5.          D:\
```

the `WindowsFileSystems.getFileStores()` report only `C:\` and `D:\` drives. Mountpouints (`Volume` label in the example) aren't reported

**Comment by dbwiddis** [comment]
Feel free to rewrite the Windows FileSystem call to use the WMI info.  I'd be happy to get rid of the Swing thread shenanigans anyway.

**Comment by dbwiddis** [comment]
[Windows WMI call](https://msdn.microsoft.com/en-us/library/aa394173(v=vs.85).aspx) should be (haven't tested) `WmiUtil.selectStringsFrom(null, "Win32_LogicalDisk",  "Name,Description,ProviderName,FileSystem,Freespace,Size", null);`  

Also if you want to rework the LinuxFileSystem, it appears that the Java implementation simply parses `/proc/mount` which we also do to store the filesystem type... may as well collapse that into a single loop as well.

**Comment by henryx** [comment]
Sadly, `Win32_LogicalDisk` doesn't do the same thing, because, as `Win32_DiskDrive`, it doesn't report volumes mounted as mountpoint, I'll see an alternative (or I split code for Windows XP/2003 and Windows Vista and newer)

**Comment by dbwiddis** [comment]
Are you still working on this?

Did your previous PR give you the data you need?  I can accept your PR and then add the network drive info myself...

**Comment by henryx** [comment]
Yes, I've resumed work today, I'm sorry if I'm disappeared, but I had to do
and I could not finish it

2016-05-31 18:35 GMT+02:00 dbwiddis <notifications@github.com>:

**Comment by dbwiddis** [comment]
Fixed in #176
